lib/logger.js

```diff
@@ -11,8 +11,44 @@ Hoek.assert(process.env.LOG_LEVEL, 'Must define LOG_LEVEL');
 var stream = new PrettyStream();
 stream.pipe(process.stdout);
 
+function requestSerializer(req) {
+  return {
+    path: req.path,
+    method: req.method,
+    headers: req.headers
+  };
+}
+
+function responseSerializer(res) {
+  return {
+    payload: res.output.payload
+  };
+}
+
+function errorSerializer(res) {
+  if (!res.data) {
```

Comment:
In the log, just send `req.data.error` directly so that you don't have to do any of these checks